### ISSUE TITLE
No sound after moving cursor to textbox--default-story field

### ISSUE THREAD

**Issue by AgnieszkaRadecka** [Sage/carbon#5169]
### Current behaviour

1. Go to https://carbon.sage.com/?path=/story/textbox--default-story
2. Turn on NVDA tool
3. Move cursor to text field

There is no information about text filed

### Expected behaviour

I should hear about text field

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/story/textbox--default-story

### JIRA Ticket (Sage Only)

SBS-32304

### Carbon Version

106.6.5

### What browsers are you seeing the problem on?

Chrome

### What Operating System are you seeing the problem on?

Windows

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by nicktitchmarsh** [comment]
FE-5181

**Comment by robinzigmond** [comment]
Hello @AgnieszkaRadecka - I am unable to reproduce this issue. Testing with NVDA on Chrome on Windows, with Carbon v 106.6.5, when I move focus to the textbox - either by clicking or by using the keyboard - NVDA announces "textbox, edit selected. textbox". Could you please give more details about the precise issue you've observed?

**Comment by AgnieszkaRadecka** [comment]
Hi, yeah... it doesn't occur anymore
I ll close it
